dali/core/geom_transform_test.cc

```diff
@@ -66,6 +66,68 @@ TEST(GeomTransform, Rotation2D) {
   EXPECT_NEAR(rotated4.y, 0,  1e-6f);
 }
 
+struct vec_near {
+  template <int N, typename T>
```

Comment:
this only makes sense for floating point types, right?

Reply:
In theory, you can have some tolerance even for integer types.